templates/scenarios/js/workflow/teamsfx/app.local.yml

```diff
@@ -0,0 +1,48 @@
+version: 1.0.0
+
+registerApp:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: notification-bot # Teams app name
```

Comment:
Seems we need to use a placeholder for teams app name {%appName%}

Reply:
thanks. Will remove local yml file as local debug is not ready.